src/main/java/com/synopsys/integration/alert/channel/jira/common/JiraTestIssueRequestCreator.java

```diff
@@ -96,7 +96,7 @@ public IssueTrackerRequest createRequest(IssueOperation operation, String messag
 
     private ProviderMessageContent createTestNotificationContent(ItemOperation operation, String messageId, String customTopic, String customMessage) throws AlertException {
         return new ProviderMessageContent.Builder()
-                   .applyProvider("Alert")
+                   .applyProvider("Alert", -1L)
```

Comment:
We should be able to get a real value for this at this point.

Reply:
This is just creating a test Issue for Jira.  We aren't using any provider specific data at this point to create a test issue.